pkg/configurations/configurations.go

```diff
@@ -201,11 +202,11 @@ func (c *configurations) getDockerComposeConfigurations(scope string) []Configur
 				Age:             age.GetAge(info.ModTime()),
 				TracingEnabled:  tracingEnabled(comp.Spec.TracingSpec.SamplingRate),
 				SamplingRate:    comp.Spec.TracingSpec.SamplingRate,
-				MetricsEnabled:  comp.Spec.MetricSpec.Enabled,
+				MetricsEnabled:  *comp.Spec.MetricSpec.Enabled,
 				MTLSEnabled:     comp.Spec.MTLSSpec.Enabled,
 				WorkloadCertTTL: comp.Spec.MTLSSpec.WorkloadCertTTL,
 				ClockSkew:       comp.Spec.MTLSSpec.AllowedClockSkew,
-				Manifest:        content,
+				Manifest:        comp.String(),
```

Comment:
Why this change here?

Reply:
Maybe Im misinterpreting things, but pre bump `config.LoadStandaloneConfiguration(path)` returned the content as string together with the actual component. Now, after bump, this return param has gone so I needed to use the string method. Am I overlooking something?

Reply:
Ok thanks for pointing that out!